.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.8'
+
+services:
+  devcontainer:
+    image: mcr.microsoft.com/vscode/devcontainers/base:debian@sha256:f1ac6335fc8cdd253c673a76edf77d81b5aa4bf9746bc67ecce4337d6bbaeb20
+    command: sleep infinity
+    depends_on:
+      - dynamodb
+      - redis
+    ports:
+      - 3001:3001
+    volumes:
+      - ..:/workspace/forms-api:cached
+    environment:
+      AWS_REGION: ca-central-1
+      AWS_ACCESS_KEY_ID: AWSACCESSKEYID
+      AWS_SECRET_ACCESS_KEY: AWSSECRETACCESSKEY
+
+  dynamodb:
```

Comment:
Interesting! How will this work with the application? Should SDK clients be initialized a certain way to communicate with this container? Do you have a way to access the DynamoDB container to insert fake data in it?

Reply:
Yup, that's exactly it - you update the SDK endpoint to reference the `dynamodb` service so it would become `http://dynamodb:8000` from within the devcontainer network.  You then need to manually create the tables and insert data.

That being said I just found out that I should be able to get a pro license soon, so this PR can be closed since it'll be easier to run the same setup as everyone else.